VideoWeb/VideoWeb/ClientApp/src/app/security/idp-selection/idp-selection.component.ts

```diff
@@ -23,42 +25,51 @@ export class IdpSelectionComponent implements OnInit {
     submitted = false;
 
     constructor(
-        private adalSvc: AdalService,
+        private oidcSecurityService: OidcSecurityService,
         private route: ActivatedRoute,
         private router: Router,
         private returnUrlService: ReturnUrlService,
         private logger: Logger
     ) {}
 
     ngOnInit(): void {
-        if (this.isLoggedIn()) {
-            const returnUrl = this.returnUrlService.popUrl() || '/';
-            try {
-                this.logger.debug(`[IdpSelectionComponent] - User is authenticated. Returning to ${returnUrl}`);
-                this.router.navigateByUrl(returnUrl);
-            } catch (e) {
-                this.logger.error('[IdpSelectionComponent] - Failed to log in', e);
-                this.router.navigate(['/']);
-            }
-        } else {
-            const returnUrl = this.route.snapshot.queryParams['returnUrl'] || '/';
-            this.returnUrlService.setUrl(returnUrl);
+        this.oidcSecurityService
+            .checkAuth()
+            .pipe(
+                catchError(err => {
+                    this.logger.error('*** [Idp Selection] - Check Auth Error', err);
+                    this.router.navigate(['/']);
+                    return NEVER;
+                })
+            )
+            .subscribe(loggedIn => {
+                this.logger.debug('*** [IdpSelectionComponent] - isLoggedIn ' + loggedIn);
```

Comment:
I think we should update the console logger to not write debug messages to the console in production mode

Reply:
removed the *** changed the comments to be less debuggy also.